src/c.rs

```diff
@@ -41,17 +41,6 @@ macro_rules! define_metrics_tests {
       $expected_align_factor:expr ) =>
     {
         #[cfg(test)]
-        extern {
```

Comment:
Sorry, I'm not sure I understand. Did you try defining `GFp_unsigned_long_align` and `GFp_unsigned_long_size` in crypto/crypto.c and using them?